jobs/run_collect.sh

```diff
@@ -79,18 +80,6 @@ if [ "$IS_DEBUG_JOB" != "yes" ]
                 echo "Removing HyperV temporary console logs.."
                 rm -fv /home/jenkins-slave/logs/hyperv-build-log-$ZUUL_UUID-$hyperv01
                 rm -fv /home/jenkins-slave/logs/hyperv-build-log-$ZUUL_UUID-$hyperv02
-
-		echo "Releasing devstack floating IP"
-		nova remove-floating-ip "$VMID" "$FLOATING_IP"
```

Comment:
Because with the new workflow we cleanup in a separate script, aka run_cleanup.sh